### DISCUSSION TITLE
How to solved  ReactDOM.render is no longer supported in React 18 for react-native ? I know solution for react application

### DISCUSSION THREAD

**Discussion by Alomgir27** [community/community#31091] [Other Feature Feedback, Questions, & Ideas]
error messages
index.js:1 Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
console.<computed> @ index.js:1

**Comment by melvincwng** [comment]
Hi @Alomgir27, to resolve that error message in React 18, in your index.js file - kindly change to the following code below: 

```
import React from "react";
import ReactDOM from "react-dom";
import "./index.css";
import App from "./App";
import reportWebVitals from "./reportWebVitals";

const root = ReactDOM.createRoot(document.getElementById("root"));
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>
);

reportWebVitals();
```

Hope this answers your question and resolves your issue 😊

**Reply by Alomgir27** [reply]
@melvincwng  Hi, I know this, But the problem is Where should I have to add this in react-native project ?
